plugins/catalog-backend/src/database/search.ts

```diff
@@ -28,16 +28,18 @@ const SPECIAL_KEYS = [
   'metadata.generation',
 ];
 
-function toValue(current: any): string | null {
-  if (current === undefined || current === null) {
-    return null;
-  }
+// The maximum length allowed for search values. These columns are indexed, and
+// database engines do not like to index on massive values. For example,
+// postgres will balk after 8191 byte line sizes.
+const MAX_VALUE_LENGTH = 200;
 
-  return String(current).toLowerCase();
-}
+type Kv = {
+  key: string;
+  value: any;
```

Comment:
`unknown` is better if it works

Reply:
ah yeah totally, fixed

Reply:
wuuut i did a derpo, will have to put it in the next pr